push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java

```diff
@@ -0,0 +1,74 @@
+package org.jboss.aerogear.unifiedpush.message;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.jboss.aerogear.unifiedpush.api.Variant;
+import org.jboss.aerogear.unifiedpush.dao.BatchException;
+import org.jboss.aerogear.unifiedpush.dao.ResultsStream;
+import org.jboss.aerogear.unifiedpush.message.jms.Dequeue;
+import org.jboss.aerogear.unifiedpush.message.jms.DispatchToQueue;
+import org.jboss.aerogear.unifiedpush.service.ClientInstallationService;
+import org.jboss.aerogear.unifiedpush.utils.AeroGearLogger;
+
+public class TokenLoader {
+
+    private final static int NUMBER_OF_BATCHES = 10;
+    private final static int BATCH_SIZE = 1000;
```

Comment:
Customizations for each queue will come in [AGPUSH-1357](https://issues.jboss.org/browse/AGPUSH-1357)

This is a sensible default configuration that will work for all push networks.

I wonder if we should allow to configure it even in alpha release with e.g. properties file?